src/widgets/Window.cpp

```diff
@@ -69,6 +69,26 @@ Window::Window(WindowType type)
         getSettings()->tabDirection.connect([this](int val) {
             this->notebook_->setTabDirection(NotebookTabDirection(val));
         });
+
+        actionExit_ = new QAction("Exit", this);
+        actionShow_ = new QAction("Show", this);
+        trayContextMenu_ = new QMenu(this);
+
+        trayContextMenu_->addAction(actionShow_);
+        trayContextMenu_->addSeparator();
+        trayContextMenu_->addAction(actionExit_);
+
+        connect(actionExit_, SIGNAL(triggered()), this, SLOT(exitAction()));
+        connect(actionShow_, SIGNAL(triggered()), this, SLOT(showAction()));
+
+        trayIcon_ = new QSystemTrayIcon(this);
+        trayIcon_->setIcon(QIcon(":/icon.ico"));
+        trayIcon_->setToolTip("Chatterino");
```

Comment:
I'd modify the tooltip to be more consistent with the window title and at least include version.